samples/android-app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Is this the "normal" proguard directory? Don't project config things normally go in a `config` file?
Eg: `config/checkstyle`, `config/proguard`?

Reply:
Yes, this default android project structure.